sparkmagic/sparkmagic/livyclientlib/reliablehttpclient.py

```diff
@@ -94,3 +96,8 @@ def _send_request_helper(self, url, accepted_status_codes, function, data, retry
                     raise HttpClientException(u"Invalid status code '{}' from {} with error payload: {}"
                                               .format(status, url, text))
             return r
+
+    def _set_custom_certfiles_path(self):
+        if conf.custom_certfiles_path is not None:
```

Comment:
`conf.custom_certfiles_path` is not a method call. This needs to invoke the method to retrieve the value of the configuration. So it would change to `conf.custom_certfiles_path()`. That's what I meant with my previous comment 😃

Reply:
So just put this method in line 37, like you had it before, but make sure to actually retrieve the value of the configuration. Does that make sense?